ratecoeff.cc

```diff
@@ -44,7 +45,8 @@ typedef struct {
 
 static char adatafile_hash[33];
 static char compositionfile_hash[33];
-static char phixsfile_hash[33];
+static char phixsfile_v1_hash[33];
+static char phixsfile_v2_hash[33];
```

Comment:
Put these in an array similar to phixsdata_filenames for easy looping. Also add an array for phixs_file_version_exists.

Reply:
having trouble initialising the phixsfile_hash_array, I've tried a lot of different variations of this but none have worked:

__managed__ std::array<char 33, 3> phixsfile_hash;

I'm trying to make an array 3 elements long all with all elements having 32 characters

Reply:
You forgot the brackets around the number 33.